packages/@functionless/aws-apigateway/src/rest-api.ts

```diff
@@ -1,17 +1,17 @@
-import type * as cdk from "aws-cdk-lib";
+import type { aws_apigateway } from "aws-cdk-lib";
```

Comment:
wonder if we should start using the service specific imports with the paired down packages. Can come later, but it does reduce the amount of aws-cdk that is pulled in at any time.